### PR TITLE
Added Minified CSS using gulp-clean-css API

### PR DESCRIPTION
**Pull request by ashwinjayan**
Minified css automatically generated using gulp-clean-css added to gulpfile.js folder

### CODE REVIEW

**Comment by RunDevelopment** [comment]
Thank you for making this PR @ashwinjayan!
(You forgot to add `gulp-clean-css` as a dev dependency. Please commit the changes to `package.json` and `package-lock.json`.)

@ashwinjayan @mAAdhaTTah @LeaVerou @Golmote 
Is this something we want/need for Prism? I see that it makes sense to offer minified themes and plugin CSS but I never saw this as a priority because the file size of minified themes is only about 25% to 30% less than the unminified versions.

Note: If we added the minified files, we should also add support for this to the download page (shouldn't be too hard). However, changes to the download page shouldn't be part of this PR, IMO.

Thoughts?

---

Related issues: #2546, #956

**Comment by ashwinjayan** [comment]
Hi @RunDevelopment, I missed to save the dependency. Have added it to dev-dependency and committed.

File: gulpfile.js/paths.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ module.exports = {
 	componentsFile: 'components.json',
 	componentsFileJS: 'components.js',
 	components: ['components/**/*.js', '!components/index.js', '!components/**/*.min.js'],
+	themes: 'themes/*.css',
```
Comment (RunDevelopment) [inline gulpfile.js/paths.js:7]: Do you see the `.min.min.css` files as well?

```suggestion
	themes: ['themes/*.css', '!themes/*.min.css'],
```

Comment (ashwinjayan) [reply gulpfile.js/paths.js]: Have made this changes and committed the code again.

File: gulpfile.js/index.js [hunk 2]
```diff
@@ -209,13 +210,17 @@ async function languagePlugins() {
...
+const theme = minifyCSS;
 
 module.exports = {
 	watch: watchComponentsAndPlugins,
-	default: series(parallel(components, plugins, componentsJsonToJs, build), docs),
+	default: series(parallel(components, theme, plugins, componentsJsonToJs, build), docs),
 	linkify,
 	changes
-};
+};
```
Comment (mAAdhaTTah) [inline gulpfile.js/index.js:226]: Nit, but this should keep the newline.

**Comment by mAAdhaTTah** [comment]
@RunDevelopment How hard would it be to add to the download page? @ashwinjayan Is this something you'd be willing to tackle?

I do think it's a little curious we provide the JS minified and not the CSS so I'm open to adding this, but we should probs include the download page change as part of this.

**Comment by RunDevelopment** [comment]
> How hard would it be to add to the download page?

Pretty easy from what I see. If plugin CSS was also minified, then [this](https://github.com/PrismJS/prism/blob/0604793c64906c4307456d3cceb1b823d1ca9d9b/assets/download.js#L184) is the only line that has to be changed.

**Comment by RunDevelopment** [comment]
I'll close this now in favor of #3073.

**Pull request closed without merging**